src/main/java/ua/hillel/FileManager.java

```diff
@@ -0,0 +1,107 @@
+package ua.hillel;
+
+import java.io.*;
+import java.util.Scanner;
+import java.util.logging.Logger;
+
+public class FileManager {
+
+    private static final Logger LOGGER = Logger.getLogger(FileManager.class.getName());
+    private String currentDirectory;
+
+    public FileManager() {
+        this.currentDirectory = "/";
+    }
+
+    public void start() {
+        while (true) {
+
+            BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
+            String rawCommand;
+            try {
+                rawCommand = br.readLine();
+            } catch (IOException e) {
+                throw new RuntimeException(e);
+            }
+            Command command = parseCommand(rawCommand);
+
+            if (command != null) {
+                switch (command) {
+                    case CD -> changeDirectory(rawCommand);
+                    case LS -> listDirectory();
+                    case CP -> copyFiles(rawCommand);
+                    case PWD -> printWorkDirectory();
+                }
+            }
+        }
+    }
+
+    private Command parseCommand(String commandRaw) {
+        try {
+            if (commandRaw.contains(" ")) {
+                String subCommand = commandRaw.split(" ")[0];
+                return Enum.valueOf(Command.class, subCommand.toUpperCase());
+            } else {
+                return Enum.valueOf(Command.class, commandRaw.toUpperCase());
+            }
+
+        } catch (Exception ignore) {
+        }
+        return null;
+    }
+
+    private void changeDirectory(String command) {
+        try {
+            String subCommand = command.split(" ")[1];
+            if (subCommand.equals("..")) {
+                int lastIndex = currentDirectory.lastIndexOf("/");
+                if (lastIndex != -1) {
+                    currentDirectory = currentDirectory.substring(0, lastIndex);
+                }
+            } else {
+                currentDirectory = subCommand;
+            }
+        } catch (Exception ignore) {
+        }
+        LOGGER.info(currentDirectory);
+    }
+
+    private void listDirectory() {
+        try {
+            File file = new File(currentDirectory);
+            StringBuilder sb = new StringBuilder();
+
+            for (File f : file.listFiles()) {
+                sb.append(f.getName() + "\n");
+            }
+            LOGGER.info(sb.toString());
+        } catch (Exception e) {
+            LOGGER.info("Set correct path by using - cd");
```

Comment:
The message is not clear enough to understand what went wrong, you can change the message to something more meaningful, so the user can understand what should be fixed and pass an exception to the logger, so not only your custom message will be displayed but also a real cause of a problem